participants/Fvedio.md

```diff
@@ -0,0 +1,64 @@
+# <FVedio>
+Before submitting a PR containing your team's project information:
+
+Make sure you rename this file to contain your project's name
+Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Project Name:FVedio
+Slack: xhs
+Submission ID: 678917603
+
+## (1) Brief Project Description
+
+*Fvedio is a video community focusing on hobbies. Users can upload their own videos and share them with friends in the community.*
+
+## (2) Link to Project UI
+
+*UI link: https://www.pmdaniu.com/storages/96748/43a03fc24dc70a5f7f4079ae1dd49d18-57026/start.html#p=%E9%A6%96%E9%A1%B5&g=1&c=1*
+
+
+## (3) What does your application/UI do?
+
+*view Vedios, upload Vedios, make funny Vedios.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*GHTorrent Project、Open Images Dataset、Internet Archive and so on*
```

Comment:
Please ensure that the videos you use have a proper license to avoid any copyright issues.